src/app/services/search.service.ts

```diff
@@ -8,12 +8,28 @@ export class SearchService {
   constructor() {}
 
   results: OfferItem[];
+  searchValueForm: any;
```

Comment:
spend a moment to maybe figure out proper types for those constants. Also the function below is accepting `form:any`. But don't spend to much time on it, we will come back to it later.